packages/cw-orch-fns-derive/src/execute_fns.rs

```diff
@@ -42,7 +42,20 @@ pub fn execute_fns_derive(input: DeriveInput) -> TokenStream {
         let is_payable = payable(&variant);
         match &mut variant.fields {
             Fields::Unnamed(_) => None,
-            Fields::Unit => None,
+            Fields::Unit => {
+
+                let (maybe_coins_attr, passed_coins) = if is_payable {
+                    (quote!(coins: &[::cosmwasm_std::Coin]),quote!(Some(coins)))
+                } else {
+                    (quote!(),quote!(None))
+                };
```

Comment:
It's repeated in for named fields as well, let's move it to function!

Reply:
Fixed in #06214ce